controllers/manifest_controller.go

```diff
@@ -92,5 +92,6 @@ func ManifestReconciler(
 		declarative.WithPostRun{internalv1beta1.PostRunCreateCR},
 		declarative.WithPreDelete{internalv1beta1.PreDeleteDeleteCR},
 		declarative.WithPeriodicConsistencyCheck(checkInterval),
+		declarative.WithCRDName(internalv1beta1.GetCRDName),
```

Comment:
This `WithCRDName` is not clear enough, which CRD? Maybe rename to something like `WithModuleCRDName`